var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -95,6 +96,8 @@ class Adios2(CMakePackage, CudaPackage):
 
     # Optional language bindings, C++11 and C always provided
     variant("cuda", default=False, when="@2.8:", description="Enable CUDA support")
+    variant("kokkos", default=False, when="@2.9:", description="Enable Kokkos support")
+    variant("sycl", default=False, when="@2.10:", description="Enable SYCL support")
```

Comment:
@eugeneswalker here we only enable sycl from adios2 2.10 (not released yet)

Reply:
Missed that! Sorry! Thanks